### DISCUSSION TITLE
Week 13

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#246] [General]
My project link: https://usu-itls-x265-23-summer.github.io/Skye404Project/

HTML: intermediate
CSS: intermediate
Git: basic
Database: basic

![QR code for github website portfolio](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133051222/ceec5858-4027-4320-96e2-2bd88112f896)

**Comment by kassyoveson** [comment]
Great job Skye. Your project turned out great. I am glad that you were able to figure out the links to the pdf's. I like the look of the page and the background color you used.

**Comment by gabebott** [comment]
Hey Skye great job on your project! This site is really impressive. I think that you did a great job of attaching the links to the pdfs as well as the images. I also think that the layout and design of your site is really good. Great job!
